packages/@react-types/actiongroup/src/index.d.ts

```diff
@@ -70,5 +70,10 @@ export interface SpectrumActionGroupProps<T> extends AriaActionGroupProps<T>, St
    */
   buttonLabelBehavior?: 'show' | 'collapse' | 'hide',
   /** The icon displayed in the dropdown menu button when a selectable ActionGroup is collapsed. */
-  summaryIcon?: ReactElement
+  summaryIcon?: ReactElement,
+  /**
+   * A slot to place the ActionGroup in.
+   * @default 'actionGroup'
+   */
+  slot?: string
```

Comment:
Not entirely sure if this `slot` prop should be exposed, it all depends on our slot naming consistency and whether or not a ActionGroup could be placed in a slot other than `actionGroup`. I fine with removing this for now and adding it back in when we have a need

Same applies for ActionMenu below

Reply:
makes sense, it should be here

Reply:
Where is this slot being used by ListView?

Reply:
This slot is used here: https://github.com/adobe/react-spectrum/blob/844549837a0bfdc5943c8a6214dbd4576d027040/packages/%40react-spectrum/list/src/ListViewItem.tsx#L219-L223

Reply:
hmmmmm.... we don't havre this on any other component (other than ones designed for this, e.g. Text)? Are we sure this is needed?

Reply:
Yeah, currently the default slot provided to ActionMenu/ActionGroup suffices for now, definitely isn't needed but was prompted by the ListView stories: https://github.com/adobe/react-spectrum/blob/418e59ba5be2418cb52ea5087162a6faeeb1cc5a/packages/%40react-spectrum/list/stories/ListView.stories.tsx#L255-L261. I'll remove it for now.